http_server_test.go

```diff
@@ -163,17 +158,18 @@ func TestGetMetadataTinkerbellKant(t *testing.T) {
 	defer os.Setenv("DATA_MODEL_VERSION", dataModelVersion)
 	os.Setenv("DATA_MODEL_VERSION", "1")
 
-	customEndpoints := os.Getenv("CUSTOM_ENDPOINTS")
-	defer os.Setenv("CUSTOM_ENDPOINTS", customEndpoints)
-	os.Setenv("CUSTOM_ENDPOINTS", `{"/metadata":".metadata.instance","/components":".metadata.components","/userdata":".metadata.userdata"}`)
-
+	customEndPoints := map[string]string{
```

Comment:
super nitpicky i know, but can you lower case the the p here (`customEndpoints`)?

Reply:
:+1:

Reply:
fixed, PTAL